build.xml

```diff
@@ -50,7 +50,8 @@
             <arg path="src"/>
             <arg value="xml"/>
             <arg value="phpmd.xml"/>
-            <arg value="--exclude */Tests/*"/>
+            <arg value="--exclude"/>
```

Comment:
Check the diff. I changed it because `<arg line="--exclude */Tests/*"/>` did not work. For reference see:
- https://github.com/OpenConext/Stepup-Middleware/blob/66ecc1abc6c7858949b15274464db7b4378c01d3/build.xml#L47-L56
- https://github.com/OpenConext/Stepup-SelfService/blob/343d5ec8c9690e25a147bf0d054ff88fab4654b0/build.xml#L47-L56